build/Dockerfile

```
@@ -8,26 +18,28 @@ COPY go.mod go.mod
 COPY go.sum go.sum
 # cache deps before building and copying source so that we don't need to re-download as much
 # and so that source changes don't invalidate our downloaded layer
-RUN go mod download
+# In downstream this needs to be a cached vendor/ folder, so do same upstream
+RUN go mod tidy; go mod vendor
 
 # Copy the go source
 COPY . .
 
-# compile workspace controller binaries
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build \
+# compile workspace controller binaries, then webhook binaries
+RUN export ARCH="amd64" && \
+  CGO_ENABLED=0 GOOS=linux GOARCH=${ARCH} GO111MODULE=on go build -mod=vendor -x \
```

```suggestion
  CGO_ENABLED=0 GOOS=linux GOARCH=${ARCH} GO111MODULE=on go build -mod=vendor  \
```
Do we really need debug info output?